js/commonJs.js

```diff
@@ -0,0 +1,462 @@
+document.addEventListener("DOMContentLoaded", function (e) {
```

Comment:
You mean those two methods, right?
It happend to me couple of times that I linked the code of `addFacetWidget` method to customers because they wanted to update behaviour of some widgets. All widgets are loaded in instatnsearch.js file except for facets. That's why I would move the method - it's updated by customers and "internals" should not be customized.

`getAutocompleteSource` is not that obvious, true.